developer-docs/latest/concepts/sso.md

```diff
@@ -0,0 +1,320 @@
+              ],
+              callbackURL: strapi.admin.services.passport.getStrategyCallbackURL('google')
+            }, (request, accessToken, refreshToken, profile, done) => {
+              done(null, {
+                email: profile.email,
+                firstname: profile.given_name,
+                lastname: profile.family_name,
+              });
+            }
+          ),
+        },
+      ],
+    },
+  },
+});
+```
+
+:::
+
+::: tab Github
+
+Using: [passport-github](https://github.com/cfsghost/passport-github)
+
+```bash
+yarn add passport-github2
+```
+
+`/config/server.js`
+
+```jsx
+'use strict';
+
+const GithubStrategy = require('passport-github2');
+
+module.exports = ({ env }) => ({
+  // ...
+  admin: {
+    // ...
+    auth: {
+      // ...
+      providers: [
+        {
+          uid: 'github',
+          displayName: 'Github',
+          icon: 'https://cdn1.iconfinder.com/data/icons/logotypes/32/github-512.png',
+          createStrategy: strapi => new GithubStrategy({
+            clientID: env('GITHUB_CLIENT_ID'),
+            clientSecret: env('GITHUB_CLIENT_SECRET'),
+            scope: ['user:email'],
+            callbackURL: strapi.admin.services.passport.getStrategyCallbackURL('github'),
+          }, (accessToken, refreshToken, profile, done) => {
+            done(null, {
+              email: profile.emails[0].value,
+              username: profile.username,
+            });
+          }),
+        },
+      ],
+    },
+  },
+});
+```
+:::
+
+::: tab Discord
+
+Using: [passport-discord](https://github.com/nicholastay/passport-discord#readme)
+
+```bash
+yarn add passport-discord
+```
+
+`/config/server.js`
+
+```jsx
+"use strict";
+
+const DiscordStrategy = require("passport-discord");
+
+module.exports = ({ env }) => ({
+  // ...
+  admin: {
+    // ...
+    auth: {
+      // ...
+      providers: [
+        {
+          uid: 'discord',
+          displayName: 'Discord',
+          icon: 'https://cdn0.iconfinder.com/data/icons/free-social-media-set/24/discord-512.png',
+          createStrategy: strapi =>
+            new DiscordStrategy(
+              {
+                clientID: env('DISCORD_CLIENT_ID'),
+                clientSecret: env('DISCORD_SECRET'),
+                callbackURL: strapi.admin.services.passport.getStrategyCallbackURL(
+                  'discord'
+                ),
+                scope: ['identify', 'email'],
+              },
+              (accessToken, refreshToken, profile, done) => {
+                done(null, {
+                  email: profile.email,
+                  username: `${profile.username}#${profile.discriminator}`,
+                });
+              }
+            ),
+        },
+      ],
+    },
+  },
+});
+```
+:::
+::::
+
+## Advanced Customization
+
+### Admin Panel URL
+
+If your administration panel lives on a different host/port than your Strapi server, you will need to modify the admin URL.
```

Comment:
Not always the case, this is anytime it's sitting behind a proxy, which will be almost always if you need SSL.

This will need to be reworded.

Reply:
Do you have a suggestion?

Reply:
As we're about to release this feature, do you have any idea/tip about how to reword this part @derrickmehaffy?

Reply:
:thinking: it really depends on the deployment, I would say it would only apply if you are using sub-folder based proxying. If you are just using a sub-domain then you don't need to set the admin url.

See the examples for Nginx: https://strapi.io/documentation/developer-docs/latest/deployment/nginx-proxy.html#strapi-server

Reply:
The only example that needs this var set is on the sub-folder split config